app/Console/Commands/SendReminderEmailsCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Enums\UserMailPreference;
+use App\Mail\PendingNotifications;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Mail;
+
+final class SendReminderEmailsCommand extends Command
```

Comment:
Feels that this should not be called "reminder".

Reply:
`SendUnreadNotificationEmailsCommand`?

Reply:
@CamKem any suggestions?